packages/itwin/tree-widget/src/test/trees/models-tree/ModelsTreeComponent.test.tsx

```diff
@@ -245,6 +271,14 @@ describe("<ModelsTreeComponent />", () => {
     });
 
     describe("<View3DButton />", () => {
+      it("renders enlarged View3DButton", async () => {
+        const { getByRole } = render(
+          <ModelsTreeComponent.View3DButton models={[{ id: "modelTestId", isPlanProjection: true }]} viewport={mockViewport().object} density="enlarged" />,
+        );
+        const button = await waitFor(() => getByRole("button"));
+        expect(button.getAttribute("data-iui-size")).to.be.null;
```

Comment:
Don't thin we can count on `data-iui-size` allways being there. It looks like internal implementation detail of itwinui-react. 

I think enlarged layout is better tested with e2e so for getting coverage up I think we can render `enlarged` button in one of the other tests.

Reply:
Removed the tests and added `enlarged` buttons to other tests.